src/devices/Bmxx80/Bme680.cs

```diff
@@ -0,0 +1,235 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.I2c;
+using Iot.Device.Bmxx80.CalibrationData;
+using Iot.Device.Bmxx80.PowerMode;
+using Iot.Device.Bmxx80.Register;
+using Iot.Units;
+
+namespace Iot.Device.Bmxx80
+{
+    /// <summary>
+    /// Represents a BME680 temperature, pressure, relative humidity and VOC gas sensor.
+    /// </summary>
+    public class Bme680 : Bmxx80Base
+    {
+        /// <summary>
+        /// Gets a value indicating whether new data is available.
+        /// </summary>
+        public bool HasNewData => ReadHasNewData();
```

Comment:
In order to be a bit more similar compared to the 280 devices, can we instead of having a property accesor to instead have ReadHumidity, ReadPreassure and all the rest public and remove these properties? It would actually be great if we could just have an internal interface that all devices implemented so that way they have a shared API design. What do you think?

Reply:
Something like `ITemperature`, `IPressure` `IHumidity`?

Reply:
@georgemathieson I've already tried: #256 - I still think we will eventually end up doing it but it is better to spend a bit more time trying to figure out if we need some more data structures (i.e. we have alredy added Temperature struct since then) - once people start using interfaces it will be hard to make any changes to them